socs/agents/ucsc_radiometer/agent.py

```diff
@@ -0,0 +1,122 @@
+from os import environ
+
+import requests
+import txaio
+from ocs import ocs_agent, site_config
+from ocs.ocs_twisted import Pacemaker, TimeoutLock
+
+
+class UCSCRadiometerAgent:
+    """Monitor the PWV Flask Server.
+
+    Parameters
+    ----------
+    agent : OCS Agent
+        OCSAgent object which forms this Agent
+    url : str
+        url of the flask server on the internet
+    year : int
+        year for the corresponding Julian Day
+    """
+
+    def __init__(self, agent, url, year):
+        self.agent = agent
+        self.log = agent.log
+        self.lock = TimeoutLock()
+
+        self.url = url
+        self.year = year
```

Comment:
`self.year` never gets used. Remove from this class as well as from the Agent arguments below.